src/transport/PeerMessageCounter.h

```diff
@@ -108,7 +169,16 @@ class PeerMessageCounter
         return CHIP_NO_ERROR;
     }
 
-    CHIP_ERROR VerifyOrTrustFirst(uint32_t counter)
+    CHIP_ERROR Verify(uint32_t counter, bool useGroupAlgorithm = false)
```

Comment:
Why do we need this helper method?

Reply:
For MCSP once implemented and to keep the previous logic intact. Using VerifyOrTrustFirst when trusting first otherwise using the Verify method for mcsp. This also prevent distinction between Unicast method called and Group method called.